app/views/option_sets/_form.html.erb

```diff
@@ -17,7 +17,7 @@
     <%= f.field(:geographic, :type => :check_box, :read_only => cannot?(:change_geographic, @option_set)) %>
     <%= f.field(:allow_coordinates, :type => :check_box, :read_only => cannot?(:change_geographic, @option_set)) %>
     <%= f.field(:multi_level, :type => :check_box,
-      :read_only => cannot?(:update_core, @option_set) || @option_set.has_select_multiple_questions? || @option_set.huge?) %>
+      :read_only => cannot?(:update_core, @option_set) || @option_set.has_select_multiple_questions? || @option_set.huge? || params[:question_type] == "select_multiple") %>
```

Comment:
Yikes this is getting a bit hairy. How about this:
1. Add an attr_accessor called `adding_to_question_type` to OptionSet with a comment of 'Indicates that this OptionSet is being created to be added to a question of the given type'.
1. Change the GET param to match this.
1. In OptSetController, do `OptionSet.new(adding_to_question_type: params[:adding_to_question_type])` 
1. Add a new method to OptionSet called `can_be_multilevel?` that checks has_select_multiple_questions, huge, and adding_to_question_type.
1. Call that in the template.